Libplanet.Headless/Hosting/LibplanetHostingExtensions.cs

```diff
@@ -65,7 +65,16 @@ IImmutableSet<Currency> nativeTokens
         });
         services.AddSingleton<BlockChain<T>>();
         services.AddSingleton(_ => configuration);
-        services.AddHostedService<SwarmService<T>>();
+
+        Peer[] peers = configuration.PeerStrings is { } ? configuration.PeerStrings.
+            Select(BoundPeer.ParsePeer).ToArray() : new Peer[] { };
+
+        services.AddHostedService(provider =>
+            new SwarmService<T>(
```

Comment:
it seems a little bit tricky because we want to limit object creation and lookup directly(i.e., `.GetRequiredService()`) in general. (see also: https://docs.microsoft.com/en-us/dotnet/core/extensions/dependency-injection-guidelines#services-not-created-by-the-service-container)

But on the other hands, planet-node doesn't have affordable options for it yet and a lot of existing services already had been created by manual. 😅 

So this implementation is okay at this moment and we need to study for suitable DI patterns in near future.